var/spack/repos/builtin/packages/py-psutil/package.py

```diff
@@ -15,12 +15,17 @@ class PyPsutil(PythonPackage):
     url      = "https://pypi.io/packages/source/p/psutil/psutil-5.6.3.tar.gz"
 
     version('5.6.3', sha256='863a85c1c0a5103a12c05a35e59d336e1d665747e531256e061213e2e90f63f3')
+    version('5.6.2', sha256='828e1c3ca6756c54ac00f1427fdac8b12e21b8a068c3bb9b631a1734cada25ed')
     version('5.5.1', sha256='72cebfaa422b7978a1d3632b65ff734a34c6b34f4578b68a5c204d633756b810')
     version('5.4.5', sha256='ebe293be36bb24b95cdefc5131635496e88b17fabbcf1e4bc9b5c01f5e489cfe')
     version('5.0.1', sha256='9d8b7f8353a2b2eb6eb7271d42ec99d0d264a9338a37be46424d56b4e473b39e')
 
+    # variant('enum', default=False) # doesn't seem to do anything...
+    # conflicts('+enum', when='@:5.3.0')
+
     depends_on('python@2.6:2.8,3.4:', type=('build', 'run'))
     depends_on('py-setuptools', type='build')
     depends_on('py-unittest2', when='^python@:2.6', type='test')
     depends_on('py-mock', when='^python@:2.7', type='test')
     depends_on('py-ipaddress', when='^python@:3.2', type='test')
+    # depends_on('py-enum34', when='+enum ^python@:3.3', type=('build', 'run'))
```

Comment:
I would delete the variant and use:
```suggestion
    depends_on('py-enum34', when='^python@:3.3', type=('build', 'run'))
```
enum is imported in several files, so I think it's always needed, it's not optional.

Reply:
@adamjstewart unless I missed something, all `import enum`s are guarded with `if sys.version_info >= (3, 4): `. Also, look s like the `enum` extra is just a forgotten leftover: https://github.com/giampaolo/psutil/issues/469#issuecomment-73581137